examples/rag_indexing_query/README.md

```diff
@@ -0,0 +1,59 @@
+# RAG example: indexing and querying with Elasticsearch
+
+This example will show how you can deploy with Hayhooks an index pipeline and a query pipeline, using Elasticsearch as the document store.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+
+## 1. Create a virtual environment
+
+It's always a good idea to create a virtual environment to install the dependencies in.
+
+```bash
+python -m venv .venv
+source .venv/bin/activate
+```
+
+## 2. Install the dependencies
+
+We'll need to install the Hayhooks package and the other dependencies needed for the pipelines.
+
+```bash
+pip install -r requirements.txt
+```
+
+## 3. Launch Hayhooks
+
+For simplicity, we'll use the Hayhooks CLI to launch the Hayhooks server.
+
+```bash
+hayhooks run
+```
+
+You can achieve the same result by running [a Docker image with Hayhooks already installed](https://github.com/deepset-ai/hayhooks-open-webui-docker-compose).
+
```

Comment:
missing section on running Elasticsearch using Docker compose

Reply:
Thanks, added it!